### PR TITLE
Fix rank page issue

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
# Rank Page fixes

_There are some bunch of pages on the rank pages that show "coming soon" which needs to be resolved, for the pages to show the proper information.

fixes https://github.com/EveripediaNetwork/issues/issues/1396

### CODE REVIEW

File: src/App/marketCap/marketcapIds.ts [hunk 1]
```diff
@@ -164,4 +164,17 @@ export const cryptocurrencyIds = [
   { coingeckoId: 'ripple', wikiId: 'xrp' },
   { coingeckoId: 'tron', wikiId: 'trontrade-1' },
   { coingeckoId: 'uniswap', wikiId: 'uni' },
+  { coingeckoId: 'edgecoin-2', wikiId: 'edgecoin' },
+  { coingeckoId: 'elrond-erd-2', wikiId: 'egld' },
+  { coingeckoId: 'rocket-pool', wikiId: 'rpl' },
+  { coingeckoId: 'theta-token', wikiId: 'theta-network' },
+  { coingeckoId: 'blockstack', wikiId: 'stx-token' },
+  { coingeckoId: 'tezos', wikiId: 'tezos-xtz' },
+  { coingeckoId: 'polkadot', wikiId: 'polkadot-new' },
```
Comment (AshakaE) [inline src/App/marketCap/marketcapIds.ts:173]: I'm not sure we have a wiki `polkadot-new`

**Review by AshakaE — CHANGES_REQUESTED** [review]
looks good so far 👍 , just one minor change

**Comment by Softdev1** [comment]
SCR-20230530-nxig
@Emmanueldmlr

File: src/App/marketCap/marketcapIds.ts (hunk 1 above)
Comment (Emmanueldmlr) [reply src/App/marketCap/marketcapIds.ts]: Removed now

**Review by AshakaE — APPROVED** [review]

**Pull request merged**